### PR TITLE
Change device size to bytes

### PR DESCRIPTION
**Pull request by VadimZezyotkoATP**
Изменил размерность устройств в Байты.

### CODE REVIEW

File: docs/user_manual/ReadMe.md [hunk 1]
```diff
@@ -1142,44 +1142,45 @@ G_IOL_8 | **IO-Link** блок питания на 8 канала с автом
...
-F | PXC.2910411 | 4 | 1.5 (_12 bytes_)
-VC | BURKERT.8694 | 5 | 2.5 (_20 bytes_)
-HLA | PXC.1191993 | 0 | 1
+V | AL.9615-4003-06 | 4 | 1
+V | AL.9615-4003-08 | 4 | 1
+V | AL.9615-4004-20 | 4 | 1
+V | DEF.SORIO-1SV | 4 | 1
+FQT | SMR12GGXFRKG/US-100 | 8 | 0
+QT | IFM.LDL100 | 9 | 0
+QT | IFM.LDL200 | 9 | 0
```
Comment (KirillGutyrchik) [inline docs/user_manual/ReadMe.md:1153]:
```suggestion
QT | IFM.LDL100 | 12 | 0
QT | IFM.LDL200 | 12 | 0
```

File: docs/user_manual/ReadMe.md [hunk 2]
```diff
@@ -1142,44 +1142,45 @@ G_IOL_8 | **IO-Link** блок питания на 8 канала с автом
...
+LS | IFM.LMT104 | 2 | 0
+LS | IFM.LMT105 | 2 | 0
+LS | E&H.FTL33-GR7N2ABW5J | 2 | 0
+DEV_VTUG и Y | FES.VTUG-10-MSDR-B1Y-25V20-G18FD-DTFD-M7SFD-12K+M1SCVA | 0 | 4
+DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-12K+SCVA | 0 | 4
+DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-16K+SCVA | 0 | 4
+DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-8K+SCVA | 0 | 2
+DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FDL-DTFDL-M7SFD-4K+SCVA | 0 | 2
+F | PXC.2910411 | 8 | 1.5
+VC | BURKERT.8694 | 10 | 2.5
```
Comment (KirillGutyrchik) [inline docs/user_manual/ReadMe.md:1181]:
```suggestion
F | PXC.2910411 | 8 | 3
VC | BURKERT.8694 | 10 | 5
```

**Comment by KirillGutyrchik** [comment]
Hey @VadimZezyotkoATP, while reviewing your PR, I'd suggest the following code changes:

👉 [**Code Suggestion for #1405**](https://gitkraken.dev/drafts/6880141d-f069-42b7-a4c7-9ce0518c11fa?source=PRComment)
>https://github.com/savushkin-r-d/EasyEPLANner/pull/1405

You can also review and apply these suggestions [locally on your machine](https://gitkraken.dev/link/drafts/6880141d-f069-42b7-a4c7-9ce0518c11fa?type=suggested_pr_change&prEntityId=WyJnaXRodWIiLCJwciIsIjEiLCIiLCJQUl9rd0RPRFp2SFNzNTFmXzBEIl0=&source=PRComment).

<details><summary>Learn more about GitKraken Code Suggest</summary>

>[Code Suggest](https://gitkraken.dev/redirect/aHR0cHM6Ly93d3cuZ2l0a3Jha2VuLmNvbS9zb2x1dGlvbnMvY29kZS1zdWdnZXN0P3NvdXJjZT1naXRodWIuY29tJm1lZGl1bT1wcl9jb21tZW50?source=PRSuggest&event=redirectToCodeSuggestPage&provider=github) liberates your code reviews from GitHub's restrictive, comment-only feedback style. As simple as suggesting changes in a Google-doc, provide real code suggestions from where you code, e.g. your IDE, and on anything in your project — not just on the lines of code changed in the PR.

_Join your team on [GitKraken](https://gitkraken.dev/redirect/aHR0cHM6Ly93d3cuZ2l0a3Jha2VuLmNvbT9zb3VyY2U9Z2l0aHViLmNvbSZtZWRpdW09cHJfY29tbWVudA==?source=PRSuggest&event=redirectToIndexPage&provider=github) to speed up PR review._
</details>

File: docs/user_manual/ReadMe.md
[the remark this answers is not in the document]
Comment (idzm) [reply docs/user_manual/ReadMe.md:1]: Вот как должно быть:

```md
| Тип          | Номер типа                                             | Input (Bytes) | Output (Bytes) |
| ------------ | ------------------------------------------------------ | ------------- | -------------- |
| V            | AL.9615-4003-06                                        | 4             | 1              |
| V            | AL.9615-4003-08                                        | 4             | 1              |
| V            | AL.9615-4004-20                                        | 4             | 1              |
| V            | DEF.SORIO-1SV                                          | 4             | 1              |
| FQT          | SMR12GGXFRKG/US-100                                    | 8             | 0              |
| QT           | IFM.LDL100                                             | 12            | 0              |
| QT           | IFM.LDL200                                             | 12            | 0              |
| PT           | IFM.PI2715                                             | 2             | 0              |
| PT           | IFM.PI2794                                             | 2             | 0              |
| PT           | IFM.PI2797                                             | 2             | 0              |
| PT и LT      | IFM.PM1704                                             | 4             | 0              |
| PT           | IFM.PM1705                                             | 4             | 0              |
| PT и LT      | IFM.PM1707                                             | 4             | 0              |
| PT           | IFM.PM1708                                             | 4             | 0              |
| PT           | IFM.PM1709                                             | 4             | 0              |
| PT           | IFM.IFM.PM1715                                         | 4             | 0              |
| PT           | FES.8001446                                            | 2             | 0              |
| LT           | IFM.LR2750                                             | 2             | 0              |
| TE           | IFM.TA2502                                             | 2             | 0              |
| TE           | IFM.TA2532                                             | 2             | 0              |
| TE           | IFM.TA2535                                             | 2             | 0              |
| TE           | IFM.TA2435                                             | 2             | 0              |
| TE           | E&H.TM311-AAC0BH2BBB2A1                                | 4             | 0              |
| LS           | IFM.LMT100                                             | 2             | 0              |
| LS           | IFM.LMT102                                             | 2             | 0              |
| LS           | IFM.LMT104                                             | 2             | 0              |
| LS           | IFM.LMT105                                             | 2             | 0              |
| LS           | E&H.FTL33-GR7N2ABW5J                                   | 2             | 0              |
| DEV_VTUG и Y | FES.VTUG-10-MSDR-B1Y-25V20-G18FD-DTFD-M7SFD-12K+M1SCVA | 0             | 4              |
| DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-12K+SCVA         | 0             | 4              |
| DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-16K+SCVA         | 0             | 4              |
| DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FD-DTFD-M7SFD-8K+SCVA          | 0             | 2              |
| DEV_VTUG и Y | FES.VTUG-10-VRLK-B1Y-G18FDL-DTFDL-M7SFD-4K+SCVA        | 0             | 2              |
| F            | PXC.2910411                                            | 8             | 3              |
| VC           | BURKERT.8694                                           | 10            | 5              |
| HLA          | PXC.1191993                                            | 0             | 2              |
| G            | TRIO3-PS/1AC/24DC/10/4C/IOL                            | 18            | 7              |
```

**Review by idzm — APPROVED** [review]

**Pull request merged**